build/Dockerfile

```diff
@@ -22,26 +22,36 @@ ARG BASE_URL="https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/bin
 
 USER 0
 
-RUN mkdir -p ${MAVEN_HOME} \
-    && curl -Lso /tmp/maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz \
```

Comment:
Even if Mvnd embed its own Maven version, it may be preferable to package a separate version to avoid locking the Maven dependency to the Maven Daemon one.